src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,132 @@
+package duke.ui;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * The Ui class handles interactions with the user by displaying messages and receiving input.
+ */
+public class Ui {
+
+    /**
+     * Displays a welcome message when the Duke application starts.
+     *
+     * @return The welcome message as a string.
+     */
+    public String showWelcomeMessage() {
+        return "Hello! I'm Glenda!\nWhat can I do for you?\n";
+    }
+
+    /**
+     * Displays a goodbye message when the Duke application ends.
+     *
+     * @return The goodbye message as a string.
+     */
+    public String showGoodbyeMessage() {
+        return "Bye. Hope to see you again soon!\n";
+    }
+
+    /**
+     * Displays the task added and the current total number of tasks in the list.
+     * @param task The task that was added.
```

Comment:
I think there should be an empty line between description and params?

Reply:
Thanks for pointing that out!